connector/loganomalyconnector/factory.go

```diff
@@ -0,0 +1,53 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loganomalyconnector
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/connector"
+	"go.opentelemetry.io/collector/consumer"
+)
+
+// NewFactory returns a new factory for the Metrics Generation processor.
+func NewFactory() connector.Factory {
+	return connector.NewFactory(
+		component.MustNewType("loganomaly"),
```

Comment:
Two observations about the name:
1. It's only detecting throughput anomalies and not looking at any content within the logs.
2. The same throughput analysis can apply just as well to other signals.

So maybe it shoudl be called `throughputanomaly`?